apache-hbase-mixin/dashboards/apache-hbase-cluster-overview.libsonnet

```diff
@@ -0,0 +1,1035 @@
+      id: 'merge',
+      options: {},
+    },
+    {
+      id: 'organize',
+      options: {
+        excludeByName: {
+          Time: true,
+          'Time 1': true,
+          'Time 2': true,
+          Value: true,
+          'Value #A': true,
+          'Value #B': true,
+          __name__: true,
+          '__name__ 1': true,
+          '__name__ 2': true,
+          clusterid: true,
+          'clusterid 1': true,
+          'clusterid 2': true,
+          context: true,
+          'context 1': true,
+          'context 2': true,
+          hbase_cluster: false,
+          'hbase_cluster 1': true,
+          'hbase_cluster 2': true,
+          instance: false,
+          'instance 1': false,
+          'instance 2': true,
+          isactivemaster: false,
+          'isactivemaster 1': false,
+          'isactivemaster 2': true,
+          job: true,
+          'job 1': true,
+          'job 2': true,
+          liveregionservers: true,
+          'liveregionservers 1': true,
+          'liveregionservers 2': true,
+          servername: false,
+          'servername 1': false,
+          'servername 2': true,
+          zookeeperquorum: true,
+          'zookeeperquorum 1': true,
+          'zookeeperquorum 2': true,
+        },
+        indexByName: {
+          Time: 5,
+          'Value #A': 12,
+          'Value #B': 13,
+          __name__: 6,
+          clusterid: 7,
+          context: 8,
+          hbase_cluster: 4,
+          hostname: 1,
+          instance: 2,
+          isactivemaster: 3,
+          job: 9,
+          liveregionservers: 10,
+          servername: 0,
+          zookeeperquorum: 11,
+        },
+        renameByName: {
+          Time: '',
+          deadregionservers: 'Dead server name',
+          hbase_cluster: 'Cluster',
+          hostname: 'Hostname',
+          instance: 'Instance',
+          'instance 1': '',
+          isactivemaster: 'Is master',
+          'isactivemaster 1': 'Master',
+          servername: 'Servername',
+          'servername 1': 'Servername',
+        },
+      },
+    },
+  ],
+};
+
+local alertsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'master_num_open_connections{job=~"$job", hbase_cluster=~"$hbase_cluster"}',
+      datasource=promDatasource,
+      legendFormat='{{hbase_cluster}}',
+      format='time_series',
+    ),
+  ],
+  type: 'alertlist',
+  title: 'Alerts',
+  description: 'Panel to report on the status of integration alerts.',
+  options: {
+    alertInstanceLabelFilter: '{job=~"${job:regex}", activemq_cluster=~"${activemq_cluster:regex}", instance=~"${instance:regex}"}',
+    alertName: '',
+    dashboardAlerts: false,
+    folder: '',
+    groupBy: [],
+    groupMode: 'default',
+    maxItems: 20,
+    sortOrder: 1,
+    stateFilter: {
+      'error': true,
+      firing: true,
+      noData: true,
+      normal: true,
+      pending: true,
+    },
+    viewMode: 'list',
+  },
+};
+
+local jvmMemoryUsagePanel = {
```

Comment:
@v-zhuravlev @mshahzeb I'm considering removing the non-heap component of this dashboard, as it is always reporting >90%. That would also mean probably removing the related alert.